### ISSUE TITLE
Add MeSH to Course Objective Process Needs Refinement - More Information

### ISSUE THREAD

**Issue by dartajax** [ilios/frontend#2551]
See screen shots ...

I want to add a MeSH Term to that third Course Objective.

![image](https://cloud.githubusercontent.com/assets/7493331/22344268/5dbefaa8-e3b0-11e6-95e3-da734c76a14f.png)

The screen shifts and shoots the MeSH search box up to the top of the screen (which is fine but jarring) and there is no indication that the search box has the focus or which Course Objective has been selected to add the MeSH term to it.

![image](https://cloud.githubusercontent.com/assets/7493331/22344342/ac7dd1c8-e3b0-11e6-9533-98df52865273.png)

In other words, how (as a user) can I be assured that I have selected the correct Course Objective?

**Comment by dartajax** [comment]
![image](https://user-images.githubusercontent.com/7493331/56534464-97120980-650e-11e9-82b2-41676b17418b.png)
